packages/frontity/src/utils/slot-and-fill/__tests__/__snapshots__/use-fills.test.tsx.snap

```diff
@@ -1,5 +1,54 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Slot should override the prop provided by the theme creator with the one given by the user 1`] = `
+<div>
+  This should render instead
+</div>
+`;
+
+exports[`Slot should pass any other extra prop to the Fill 1`] = `
+<div>
+  Data passed as prop
+</div>
+`;
+
+exports[`Slot should pass the data prop to the Fill 1`] = `
+<div>
+  Data passed as prop
+</div>
+`;
+
+exports[`Slot should read the data from state.source and pass it to the Fill 1`] = `
+<div>
+  This would normally be data for this route
+</div>
+`;
+
+exports[`Slot should render 2 elements with all props 1`] = `
+Array [
+  <div>
+    {
+  "name": "namespace1 - fill 5.1",
+  "data": "This would normally be data for this route"
+}
+  </div>,
+  <div>
+    {
+  "name": "namespace1 - fill 5.2",
+  "data": "This would normally be data for this route"
+}
+  </div>,
+]
+`;
+
+exports[`Slot should render children as a fallback 1`] = `"fallback"`;
+
+exports[`Slot should work in the most basic case 1`] = `
+<div>
+  Im a Fill
+</div>
+`;
+
```

Comment:
I guess the snapshots were missing because tests were failing, right? Did we merge this PR using `force` or something? Just curious. :detective:

Reply:
I have no idea. I also noticed that and I investigated a bit but didn't find any reason why those snapshots were missing 🤷